Vienna.xcodeproj/project.pbxproj

```diff
@@ -2772,6 +2772,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CODE_SIGN_ENTITLEMENTS = Vienna/ViennaDeployment.entitlements;
+				CODE_SIGN_IDENTITY = "-";
```

Comment:
Conflicts with Scripts/Resources/CS-ID.xcconfig

Reply:
Not sure what to do about this to shut Xcode up. What do you think?

Reply:
I am afraid we have to live with it for now…

Reply:
Can you try my latest commit?

Reply:
Seems fine.

Building through makefile does not work, because Sparkle's AutoUpdate.app (located in `Sparkle.framework/Versions/A/Resources`) gets an `adhoc` signature during archiving instead of a signature related to my Developer ID, but it's a separate problem.